blant.c

```diff
@@ -1173,6 +1178,103 @@ void ProcessGraphlet(GRAPH *G, SET *V, unsigned Varray[], char perm[], TINY_GRAP
 	}
 }
 
+static int nChoosek(int n, int k) {
```

Comment:
This can overflow both in the numerator and denominotor.  Please use one of these functions from libwayne, see libwayne/include/combin.h:

unsigned long long CombinChoose(int n, int m);
double CombinChooseDouble(int n, int m);